ejcIMC.py

```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    valueImc = imc
+    if imc <= 16:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez severa')
+    elif imc <= 16.99:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez moderada')
+    elif imc <= 18.49:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez aceptable')
+    elif imc <= 24.99:
+        print('Su indice de masa corporal es ',imc ,': pso normal e indica que se encuentra en con un peso adecuado')
```

Comment:
revise the print message, it say ' ; pso normal' correct the "pso"'s word

Reply:
ok, i did solve print message